packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -412,7 +412,7 @@ export class DocumentDeltaConnection
 		this.disconnect(
 			createGenericNetworkError(
 				// pre-0.58 error message: clientClosingConnection
-				"Client closing delta connection",
+				error?.message ?? "Client closing delta connection",
```

Comment:
It is very dangerous to send error.message. We only want to send very procured data.
This is why we were thinking normal/abnormal and then categorization. It is okay if that is all distilled down to single `DisconnectReason`.

Reply:
should this be "Client closing delta connection" + DisconnectReason.Unknown?